.vuepress/components/InstallInstructions.vue

```diff
@@ -155,12 +155,12 @@ usermod -a -G openhab myownuser
         &#128712; <small>	You don't need the add-ons archives if your machine has Internet access, openHAB will download add-ons you need online as necessary.</small>
         <div v-if="(selectedVersion === 'stable' || selectedVersion === 'testing')">
             <div class="download-button-container">
-              <a class="download-button" style="margin-bottom: 0" :href="addonsDownloadLink">Download openHAB {{currentDownloadVersion}} {{currentVersionLabel}} Add-ons</a>
+              <a class="download-button" style="margin-bottom: 0" :href="addonsDownloadLink" download>Download openHAB {{currentDownloadVersion}} {{currentVersionLabel}} Add-ons</a>
             </div>
         </div>
         <div v-if="selectedVersion === 'snapshot'">
             <div class="download-button-container">
-              <a class="download-button" style="margin-bottom: 0" :href="`https://ci.openhab.org/job/openHAB-Distribution/lastSuccessfulBuild/artifact/distributions/openhab-addons/target/openhab-addons-4.0.0-SNAPSHOT.kar`">Download openHAB {{$page.frontmatter.currentSnapshotVersion}} Add-ons</a>
+              <a class="download-button" style="margin-bottom: 0" :href="`https://ci.openhab.org/job/openHAB-Distribution/lastSuccessfulBuild/artifact/distributions/openhab-addons/target/openhab-addons-4.0.0-SNAPSHOT.kar`" download>Download openHAB {{$page.frontmatter.currentSnapshotVersion}} Add-ons</a>
```

Comment:
download="openhab-addons.kar">

Reply:
This would make it loose the version number in the name. The filename attribute is optional.
Can you confirm that the current version still does not work for you?

Reply:
I don't mind if you add the version number again, but I think the attribute needs to be filled so `download` has any effect at all.
In #405 I posted a screenshot which shows the issue is still there